nexus/src/app/background/tasks/support_bundle_collector.rs

```diff
@@ -867,34 +940,25 @@ impl BundleCollection {
 
         loop {
             // Process all the currently-planned steps
-            while let Some((step_name, step)) = steps.pop() {
+            while let Some(step) = steps.pop() {
                 let previous_result = tasks.spawn({
                     let collection = self.clone();
                     let dir = output.path().to_path_buf();
                     async move {
-                        debug!(collection.log, "Running step"; "name" => &step_name);
-                        step(&collection, dir.as_path()).await.inspect_err(|err| {
-                            warn!(
-                                collection.log,
-                                "Step failed";
-                                "name" => &step_name,
-                                InlineErrorChain::new(err.as_ref()),
-                            );
-                        })
+                        debug!(collection.log, "Running step"; "name" => &step.name);
```

Comment:
nit, take it or leave it: perhaps this ought to be
```suggestion
                        debug!(collection.log, "Running step"; "step" => &step.name);
```
as it seems a bit less likely to include totally unrelated logs when filtering `looker` output or similar based on a particular step?

Reply:
Updated